From: Bill Burns <bburns@redhat.com>
Date: Wed, 17 Dec 2008 08:02:47 -0500
Subject: Revert: [xen] console: make LUKS passphrase readable
Message-id: 20081217130246.4307.52242.sendpatchset@localhost.localdomain
O-Subject: [RHEL5.4 PATCH 1/2] Revert kernel fix for LUKS passphrase
Bugzilla: 475986
RH-Acked-by: Chris Lalancette <clalance@redhat.com>

Fixes bz 475986
https://bugzilla.redhat.com/show_bug.cgi?id=475986

This part of the patch is a request to remove the
following patch from the kernel spec file:

Patch23454: linux-2.6-xen-console-make-luks-passphrase-readable.patch

The problem we fixed in the kernel has been fixed upstream
in the hypervisor. This reverts the kernel fix and the
next patch supplies the hypervisor fix.

diff --git a/drivers/xen/console/console.c b/drivers/xen/console/console.c
index 0e7d088..d16096d 100644
--- a/drivers/xen/console/console.c
+++ b/drivers/xen/console/console.c
@@ -189,8 +189,6 @@ static struct console kcons_info = {
 
 static int __init xen_console_init(void)
 {
-        int i;
-
 	if (!is_running_on_xen())
 		goto out;
 
@@ -237,17 +235,6 @@ static int __init xen_console_init(void)
 
 	register_console(&kcons_info);
 
-        /*
-         * If loglevel is low assume "quiet" arg and clear
-         * the screen to remove Hypervisor messages
-         * so that user startup prompts/messages
-         * can be seen clearly. Fixes BZ 466240.
-         */
-        if ((console_loglevel <  default_console_loglevel) &&
-            (screen_info.orig_video_mode == 3)) {
-                for (i = 0; i < screen_info.orig_video_lines; i++)
-                        printk(KERN_ERR "\n");
-        }
  out:
 	return 0;
 }
